app/_data/docs_nav_konnect.yml

```diff
@@ -35,33 +35,9 @@
           url: /getting-started/publish-service
         - text: Register Applications
           url: /getting-started/app-registration
-    - text: Import Kong Gateway Entities into Konnect Cloud
+    - text: Import Kong Gateway Entities into Konnect
       url: /getting-started/import
 
-- title: Service Hub
-  icon: /assets/images/icons/konnect/icn-servicehub-nav.svg
-  items:
-    - text: Overview
-      url: /servicehub
-    - text: Manage Services
-      items:
-        - text: Konnect Services
-          url: /servicehub/manage-services
-        - text: Service Versions
-          url: /servicehub/service-versions
-        - text: Service Implementations
-          url: /servicehub/service-implementations
-        - text: Manage Service Documentation
-          url: /servicehub/service-documentation
-    - text: Manage Plugins
-      items:
-        - text: Overview
-          url: /servicehub/plugins
-        - text: Configure a Plugin on a Service
-          url: /servicehub/enable-service-plugin
-        - text: Configure a Plugin on a Route
-          url: /servicehub/enable-route-plugin
-
```

Comment:
Did you mean to move this? I'm guessing it came from the rename PR.

Reply:
This is from the rename PR. I pulled that change here so I could work from what the docs will look like once #4440 is merged.